nowide/CMakeLists.txt

```diff
@@ -1,70 +1,7 @@
-        endif()
-    endif()
+find_package(Boost 1.54 REQUIRED)
 
-    set(BOOST_NOWIDE_SKIP_TESTS ON CACHE BOOL "Disable tests in Boost.Nowide")
-
-    # Disable installing Boost.Nowide so we don't pollute the
-    # installation. We will handle installing the appropriate header files
-    # to our own vendor include dir.
-    set(BOOST_NOWIDE_SKIP_INSTALL ON CACHE BOOL "Disable installing Boost.Nowide")
-
-    # have to specify a bindir since this isn't actually a subdirectory of us
-    set(srcdir "${CMAKE_CURRENT_SOURCE_DIR}/../vendor/boost-nowide")
-    set(bindir "${CMAKE_CURRENT_BINARY_DIR}/../vendor/boost-nowide")
-    add_subdirectory(${srcdir} ${bindir})
-
-    if (LEATHERMAN_INSTALL)
-        install(DIRECTORY "${srcdir}/boost" DESTINATION "include/leatherman/vendor/boost-nowide")
-
-        if (WIN32)
-            # Have to install as FILES instead of TARGETS because the
-            # target isn't in this CMake file
-            if (LEATHERMAN_SHARED)
-                get_target_property(output_name nowide-shared OUTPUT_NAME)
-                install(FILES "${bindir}/${CMAKE_IMPORT_LIBRARY_PREFIX}${output_name}${CMAKE_IMPORT_LIBRARY_SUFFIX}" DESTINATION "lib")
-                install(FILES "${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/${CMAKE_SHARED_LIBRARY_PREFIX}${output_name}${CMAKE_SHARED_LIBRARY_SUFFIX}" DESTINATION "bin")
-            else()
-                get_target_property(output_name nowide-static OUTPUT_NAME)
-                install(FILES "${bindir}/${CMAKE_STATIC_LIBRARY_PREFIX}${output_name}${CMAKE_STATIC_LIBRARY_SUFFIX}" DESTINATION "lib")
-            endif()
-        endif()
-    endif()
-else()
-    set(${include_var} "${LEATHERMAN_PREFIX}/include/leatherman/vendor/boost-nowide")
-
-    if (WIN32)
-        if (LEATHERMAN_SHARED)
-            set(${lib_var} nowide-shared)
-            add_library(nowide-shared SHARED IMPORTED)
-            find_library(NOWIDE_LIB
-                NAMES libnowide
-                PATHS "${LEATHERMAN_PREFIX}/lib"
-                NO_DEFAULT_PATH
-                )
-            set_target_properties(nowide-shared PROPERTIES
-                IMPORTED_LINK_INTERFACE_LANGUAGES "CXX"
-                IMPORTED_LOCATION "${LEATHERMAN_PREFIX}/bin/libnowide.dll"
-                IMPORTED_IMPLIB ${NOWIDE_LIB}
-                )
-        else()
-            set(${lib_var} nowide-static)
-            add_library(nowide-static STATIC IMPORTED)
-            find_library(NOWIDE_LIB
-                NAMES libnowide
-                PATHS "${LEATHERMAN_PREFIX}/lib"
-                NO_DEFAULT_PATH
-                )
-            set_target_properties(nowide-static PROPERTIES
-                IMPORTED_LINK_INTERFACE_LANGUAGES "CXX"
-                IMPORTED_LOCATION ${NOWIDE_LIB}
-                )
-        endif()
-    endif()
+add_leatherman_includes(${Boost_INCLUDE_DIRS} "${CMAKE_CURRENT_SOURCE_DIR}/../vendor/boost-nowide")
+add_leatherman_headers(../vendor/boost-nowide/boost)
```

Comment:
`add_leatherman_headers` should munge the install location. I believe this will end up in `$PREFIX/include/leatherman/boost`? I might run it just to confirm...